src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -793,6 +793,8 @@ def load_arguments(self, _):
                    help='Gallery image version in semantic version pattern. The allowed characters are digit and period. Digits must be within the range of a 32-bit integer, e.g. <MajorVersion>.<MinorVersion>.<Patch>')
         c.argument('description', help='the description of the gallery image version')
         c.argument('managed_image', help='image name(if in the same resource group) or resource id')
+        c.argument('os_snapshot', help='Name or ID of OS disk snapshot')
+        c.argument('data_snapshot', help='Names or IDs (space-delimited) of data disk snapshots')
```

Comment:
please use nargs=+. you can find ref here https://docs.python.org/3/library/argparse.html#nargs

Reply:
It cannot be updated. Right?

Reply:
> 
> 
> please use nargs=+. you can find ref here https://docs.python.org/3/library/argparse.html#nargs

OK

Reply:
> 
> 
> It cannot be updated. Right?

image-version cannot update the source (image or snapshot) now